test/services/adsService.test.js

```diff
@@ -213,4 +213,67 @@ describe('adsService', () => {
       })
     })
   })
+
+  describe('getRandomActiveAd', () => {
+    const googleActiveAd = new Ad('Google', 'Google image').active().get()
+    const facebookDisableAd = new Ad('Facebook', 'Facebook image').disable().get()
+
+    describe('when there is zero ads', () => {
+      before(() => {
+        Database('ads').set({})
+      })
+
+      it('should return undefined', () => {
+        return AdsService().getRandomActiveAd().then(ad => {
+          expect(ad).to.equal()
```

Comment:
`to.equal(?)`... `undefined`?

Reply:
si, por que como no encuentra ad, devuelve vacío